user_guide_src/source/general/modules.rst

```diff
@@ -191,6 +191,11 @@ with the ``new`` command::
 
 Config files are automatically discovered whenever using the **config()** function that is always available.
 
+.. note:: **config()** finds the file in **app/Config/** when there is a class with the same shortname,
```

Comment:
add also on the disabling of the `$preferApp` option if this behavior is not desired.

Reply:
There is no way to do so.

Reply:
More docs are good! But we do already cover this in `Factories`: https://codeigniter.com/user_guide/concepts/factories.html#factories-options

Maybe adding a link back to there, "for more information".

Reply:
@MGatner @paulbalandan I added more explanation for `config()`.